libs/core/src/scss/base/_variables.scss

```
@@ -19,36 +19,47 @@ $sizes: (
   xxxxs: 2px,
 ) !default;
 
+// relative spacings are used sparingly when there is a need for component spacing to scale with font size
+$relative-spacings: (
```

Consider renaming this to relative-sizing, it seems like its mostly used for sizing and not spacing

---

I see that we sometimes mix up spacing and sizing. let's rename this to `$relative-sizes` so it follows the same naming as the `$sizes` variables just above, and then in the _design-tokens rename it to `--kirby-spacing-#{$name}` so it follows the same pattern as with sizes. Then internally we should refer the `utils.relative-size('s')` function. 
```suggestion
$relative-sizes: (
```

---

I've renamed it throughout, for the css custom props too, in 3bd5891